.gitignore

```diff
@@ -1,4 +1,5 @@
 *.js
 *.js.map
+*.log
```

Comment:
I strongly prefer separating functionally unrelated changes in history. Would you mind making a separate PR or at least putting it in a separate commit?

Reply:
Fair, new PR at https://github.com/lgarron/clipboard-polyfill/pull/81